src/UI/Components/Diagram/Canvas.tsx

```diff
@@ -297,7 +322,7 @@ const Canvas = ({
         }
         editable={editable}
       />
-      <CanvasWrapper id="canvas-wrapper">
+      <CanvasWrapper id="canvas-wrapper" aria-label="Composer-Container">
```

Comment:
For accessibility, adding an aria-label to a div or a span, requires you to define a role. If the aria-label was added for testing purposes, please use the data-testid instead.